packages/common/src/hardforks/shanghai.json

```diff
@@ -3,5 +3,5 @@
   "comment": "Next feature hardfork after the merge hardfork",
   "url": "https://github.com/ethereum/pm/issues/356",
   "status": "Pre-Draft",
-  "eips": []
+  "eips": [3540, 3651, 3670, 3855, 3860]
```

Comment:
I would have a strong tendency to not call this HF "shanghai" but "shandong" (or at least: a different name, I would find to re-use "shandong" here convenient) for two reasons:

1. Technically this is not the Shanghai HF but some subset or even a different set of EIPs.
2. If we use something different we are then free to merge this in and release without any fear of HF/compatibility conflicts

We can label these kind of testnet HFs in the hardfork file `comment` field with "Temporary HF for the Shandong Pre-Shanghai ephemeral testnet activating selected CFI (Considered-for-Inclusion) EIPs for the Shanghai HF".

Reply:
@holgerd77 how about we add a hardfork name : `ethereumjs` to our hardfork configs, where we can on the fly latch on the eips in our test scenarios, i.e. `Hardfork.Ethereumjs`

Reply:
Ah, no, the names for HFs for the different testnets needs to be distinct and unique, otherwise this will lead to chaos.

Maybe we go with something more explicit: "ShanghaiShandongSelection" or "ShanghaiShandongEIPs"?

A bit long, but this will also display in the client and is neither to generic (ethereumjs) nor misleading (Shanghai) nor not-graspable (Shandong).

Reply:
It is also a bit depending on if we want to merge this PR and include in a release of not.

What do you think about that question?

Reply:
I favour merge the code, and have it running it in our CI as part of new `sim` tests section

Reply:
Ok, so if we merge the code and do releases with this we should really not take anythink which we would reuse for another purpose (e.g. a follow-up testnet) with some different EIPs or so. This has some potential to get into chaos if people have different versions with e.g. Common installed.

I guess I would be in favor of a simple `Shandong` name, guess this should be graspable when seeing the whole context.

But also no need to quickly integrate if you do not yet feel comfortable with what we have, can let this sink in another 1 day or 2.

Reply:
I agree we should not call it Shanghai since we also do not match the proposed EIPs of Shanghai, so we should call it Shangdong. (We do not have EIP 4895, Beacon chain push withdrawals)